src/notification/types.js

```diff
@@ -1,4 +1,10 @@
 // @flow strict-local
+import type { UserId } from '../types';
+
+type NotificationBase = {|
+  realm_uri: string,
+  user_id: UserId,
```

Comment:
notif: Expose payload's realm_uri, present since server 2.1, to inner code

s/realm_uri/user_id/?